src/ordered.jl

```diff
@@ -56,7 +56,7 @@ struct Bayer <: AbstractOrderedDither
     level::Integer
```

Comment:
Oops this is a similar issue:

```suggestion
struct Bayer{T<:Integer} <: AbstractOrderedDither
    level::T
```